lib/foreman_ansible_core/runner/ansible_runner.rb

```diff
@@ -109,15 +111,29 @@ def write_playbook
       def start_ansible_runner
         env = {}
         env['FOREMAN_CALLBACK_DISABLE'] = '1' if @rex_command
-        command = [env, 'ansible-runner', 'run', @root, '-p', 'playbook.yml']
-        command << '--cmdline' << '"--check"' if check_mode?
-        command << verbosity if verbose?
-        initialize_command(*command)
-        logger.debug("[foreman_ansible] - Running command '#{command.join(' ')}'")
+        command = "ansible-runner run #{@root} -p playbook.yml #{cmdline} #{verbosity}"
+        initialize_command(env, command)
+        logger.debug("[foreman_ansible] - Running command '#{command}'")
+      end
+
+      def cmdline
+        cmd = [tags_cmd, check_cmd].reduce("") do |memo, item|
+          item.empty? ? memo : "#{memo} #{item}"
+        end
+        cmd.empty? ? '' : "--cmdline \"#{cmd}\""
+      end
+
+      def tags_cmd
+        flag = @tags_flag == 'include' ? '--tags' : '--skip-tags'
+        @tags.empty? ? '' : "#{flag} #{@tags}"
```

Comment:
I think that we have to accept arrays as well as strings for `@tags` (when I pass --tags="tag1,tag2" to hammer, this will be an array) and endup with `--tags ["tag1", "tag2"]` so what about:
```suggestion
        @tags.empty? ? '' : "#{flag} #{Array(@tags).join(',')}"
```